tests/templates/test_embeddings.py

```diff
@@ -34,19 +34,19 @@ class TestAmplitudeEmbedding:
     """ Tests the AmplitudeEmbedding method."""
 
```

Comment:
This looks like I accidentally blackened these tests? Sorry for that!

Reply:
Yep, the tests weren't blacked, because they have so many large matrices in them. Lots of other people have accidentally blacked the tests so far, so they are being slowly blacked by stealth 😉